### PR TITLE
Refactor 20/move discord bot

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

File: packages/bot/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "discord.js": "^12.1.1",
+    "dotenv": "^8.2.0",
+    "node-fetch": "^2.6.0"
+  },
+  "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-jest": "^23.8.2",
+    "eslint-plugin-prettier": "^3.1.3",
+    "jest": "^25.3.0",
```
Comment (hammadj) [inline packages/bot/package.json:25]: Can remove these since we have them in root

File: packages/codegen/package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "graphql-codegen-hasura-core": "^4.8.3",
     "graphql-codegen-hasura-gql": "^4.8.3",
     "graphql-codegen-hasura-react": "^4.8.3",
-    "graphql-codegen-hasura-shared": "^4.8.3",
+    "graphql-codegen-hasura-shared": "^4.8.3",
```
Comment (hammadj) [inline packages/codegen/package.json:22]: extra space

**Comment by pakokrew** [comment]
How are we doing with this PR ?

**Comment by youngkidwarrior** [comment]
im closing this for now because haven't looked at it in months

**Pull request closed without merging**